### ISSUE TITLE
Feature Request: Classified slider

### ISSUE THREAD

**Issue by ichpen** [gfnlabs/classifieds#20]
Long term would be great to see a Classi slider one could place on forum home or anywhere for that matter. Thinking of how Bob's Showcase has nice sliders, this would really work well to attract visitors.

**Comment by sikhlana** [comment]
Yes, we do have plan to add such widgets once the add-on reaches gold.
